en/theme/material/assets/css/partials/_typography.css

```
@@ -0,0 +1,61 @@
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ 
+:root {
+  --rem: 16px;
+}
+
+@font-face {
+  font-family: 'Gilmer';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Gilmer" (font-family-name-quotes)

(font-family-name-quotes)